benchmark/infrastructure/BenchmarkConfigurationShorthandVisitor.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Andre Schlegel (March of 2023, schlegea@informatik.uni-freiburg.de)
+#pragma once
+
+#include <antlr4-runtime.h>
+
+#include "../benchmark/infrastructure/generated/BenchmarkConfigurationShorthandVisitor.h"
+
+/*
+This visitor will translate the parsed short hand into a `nlohmann::json`
+object.
+*/
+class ToJsonBenchmarkConfigurationShorthandVisitor final
+    : public BenchmarkConfigurationShorthandVisitor {
+ public:
+  std::any visitShortHandString(
+      BenchmarkConfigurationShorthandParser::ShortHandStringContext* context)
+      final;
+
+  std::any visitAssignments(
+      BenchmarkConfigurationShorthandParser::AssignmentsContext* context) final;
+
+  std::any visitAssignment(
+      BenchmarkConfigurationShorthandParser::AssignmentContext* context) final;
+
+  std::any visitObject(
+      BenchmarkConfigurationShorthandParser::ObjectContext* context) final;
+
+  std::any visitList(
+      BenchmarkConfigurationShorthandParser::ListContext* context) final;
+
+  std::any visitContent(
+      BenchmarkConfigurationShorthandParser::ContentContext* context) final;
+};
```

Comment:
In the meantime (see for example the `SparqlQleverVisitor`) we use typesafe visiting. This means:
* The visitor doesn't inherit from the base class, but is its own class.
* The `visit...` methods don't return `any`, but the proper types. That way we have typesafety at compile time,
   and don't need as many `any_cast`s.

In general: Inside the visitor you can put a `using Parser = Benchmark....Parser`, then your declaarations fit in one line. The same thing you can do in the `.cpp` file.

Reply:
Done ,see [49ecb7a](https://github.com/ad-freiburg/qlever/pull/972/commits/49ecb7a1768de5ab70e8c3fba05cf652643d6c2a).